actors/user/src/main/java/org/sunbird/user/util/UserUtil.java

```diff
@@ -585,6 +586,35 @@ public static void toLower(Map<String, Object> userMap) {
               }
             });
   }
+
+  public static String[] productionUserName(String name) {
+    if (name == null || name.isEmpty()) return null;
+    // configurable blocks -- number of digits to append and numberOfUserNameRequired per method
+    // call.
+    int N = 4;
```

Comment:
Where should i put these configurable values? Should i put these in enviroment variables or external resource file.

Reply:
both env and default value in resource file

sunbird_username_num_digits